package.json

```diff
@@ -1,9 +1,12 @@
 {
     "devDependencies": {
         "@rocket.chat/apps-engine": "^1.44.0",
+        "@types/jest": "^29.5.14",
```

Comment:
This set of "prompt unit tests" are very independent of the app.

Let's not tangle up this unnecessary dependency.

Create a separate set of package.json and lock file within the prompts-library (jest) directory.